### ISSUE TITLE
how can unselect day Calendar ?

### ISSUE THREAD

**Issue by Moji0098** [mberneti/react-datepicker2#115]

**Comment by mberneti** [comment]
Hi @Moji0098 
You can set the null value [like this](https://mberneti.github.io/react-datepicker2/#example-11).

**Comment by Moji0098** [comment]
this solution empty value of box that show to user , but i want to unselect day on calendar , i want to have no selected day on calendar !

**Comment by mberneti** [comment]
@Moji0098 you can set 'setTodayOnBlur' to the 'false' value on v3.3.12.  [check this feature on the demo page](https://mberneti.github.io/react-datepicker2/#example-16)

**Comment by Moji0098** [comment]
i would not on blur today , i want onblur prev selected day :)

**Comment by mberneti** [comment]
The default value of 'setTodayOnBlur' is true for the sake of backward compatibility. And It will automatically select today. So if you want to reset value and prevent auto select today onBlur, you can set its prop to false.
(If you have a selected value, you will get it after triggering onBlur always)
Do you have any problem with the last demo example of removing the date picker value?

**Comment by Moji0098** [comment]
```
<Calendar
              ref={calendarNode}
              isGregorian={false}
              showTodayButton={false}
              onChange={...}
              value={momentJalaali(timeStampToJalali(stamp), "jYYYY/jM/jD")}
            />
```
first time when i open calendar to select day none of day is selected , **stamp** variable is null , and i select a day and trigger this code 
```
calendarNode.current.selectDay(
          momentJalaali(timeStampToJalali(stamp), "jYYYY/jM/jD")
        );
```
 and after that i change  **stamp** variable to null or ' ' or undefined value , but my calendar day be selected on preview day !! how can i do to solve this ?

**Comment by mberneti** [comment]
The momentJalaali method always returns a valid moment object (even you pass a null stamp), and you should set the value property with the null value.
By the way, there is an [example on demo](https://mberneti.github.io/react-datepicker2/#example-16) without any problem. If you reproduce this problem on the codesandbox, I can help you better.

**Comment by Moji0098** [comment]
please check this , thanks :) [example](url)

**Comment by mberneti** [comment]
@Moji0098 This issue fixed on v3.3.13 and you can set the null value on the `Calendar` component with this code:
```
calendarNode.current.selectDay(null);
```
[check it out](https://codesandbox.io/s/unruffled-pond-jikkq?file=/src/App.js:1907-1944
)
